third_party/fbjni/BUILD

```diff
@@ -45,6 +45,7 @@ cmake(
         "CMAKE_BUILD_TYPE": "$(CMAKE_BUILD_TYPE)",
         # Resolves to fbjni/jni/include
         "JNI_HEADER": "$(rlocationpath :jni_include)",
+        "CMAKE_BUILD_PARALLEL_LEVEL": "4"
```

Comment:
Is this just a broad strategy for any cmake target? My understanding was this only happened for `@hermes//:host`. If it's a broad strategy, could we pull this out to config set by command line, or set by something in `.bazelrc` to make it easily configurable to the constraints of where it's actually building?

Reply:
Yeah its broadly for every cmake target. The `@hermes//:host` target definitely needs it now but out of future looking caution, since cmake in bazel seems to be having problems identifying the correct concurrency limit I want to apply it everywhere just so we don't have to think about it. It looks like potential fixes for this are in 8 or coming in 9 so we may be able to do away with this when we are able to upgrade our bazel version.